blog/tests/models_test.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for Post model's Blog app."""
+
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from blog.models import Post
```

Comment:
Try using relative imports (I hope it should work here)

Reply:
```
from ..models import Post
```
Fixed.